### PR TITLE
Updated template shorthand

### PR DESCRIPTION
**Pull request by JeeCee1**
Changed "conditions" to "condition" (tested against 2022.11). Added link to choose. Added note about quotation marks and folding. This might be general yaml knowledge but due to all the different example I find it useful here.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by JeeCee1 — COMMENTED** [review]
Added extra comment about single and double quotation marks.

File: source/_docs/scripts/conditions.markdown [hunk 1]
```diff
@@ -436,7 +436,7 @@ Within an automation, template conditions also have access to the `trigger` vari
 ### Template condition shorthand notation
 
 The template condition has a shorthand notation that can be used to make your scripts and automations shorter. <br>
-It needs to be wrapped in quotation marks (```""```) unless using folding (```>```) where it must not use quotation marks. <br>
+It needs to be wrapped in quotation marks (```""```) unless using folding (```>```) where it must not use quotation marks. When wrapping in outside double quotation marks ensure any inner quotation marks are of the single (```'```) variant. <br>
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown:439]: I think this whole thing needs to go, it has nothing to do with the functionality in this chapter.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/scripts/conditions.markdown (hunk 1 above)
Comment (JeeCee1) [reply source/_docs/scripts/conditions.markdown:439]: See my comment in the other PR: https://github.com/home-assistant/home-assistant.io/pull/24950. Yes, beginners mistakes but there are many beginners, no?

Comment (frenck) [reply source/_docs/scripts/conditions.markdown:439]: Sure! That is why reviews are in place too. Please adjust the review comment. Thanks 👍

**Comment by frenck** [comment]
This PR is running stale; therefore going to close it for now. Feel free to re-open when ready to work on it again 👍 

../Frenck

**Pull request closed without merging**